src/test/java/org/dataloader/performance/AtomicVsAdder.java

```diff
@@ -0,0 +1,111 @@
+package org.dataloader.performance;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.LongAdder;
+
+public class AtomicVsAdder {
```

Comment:
in future a JMH benchmark is way better - we have them in dataloader

Reply:
actually, if you don't mind, I would like to drop this class as `LongAdder` is well-known class in JDK built exactly for this use case